x/wasm/types/validation.go

```diff
@@ -5,17 +5,18 @@ import (
 )
 
 const (
-	MaxWasmSize = 500 * 1024
-
 	// MaxLabelSize is the longest label that can be used when Instantiating a contract
 	MaxLabelSize = 128
 )
 
+// MaxWasmSize is the largest a compiled contract code can be when storing code on chain
+var MaxWasmSize uint64 = 800 * 1024
```

Comment:
This is good.
Can also be larger if you wish (param was upped to 1200 * 1024 earlier).

But I think 800 or 900 is a good default

Reply:
But... why make uint64 and not int? Since we compare it to length which returns an int

Reply:
I made this call because one of its uses requires a `uint64` and with split use cases, I went unsigned as a length cannot be negative. I truly don't mind changing it, though!